<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForm</class>
 <widget class="QWidget" name="SearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="0" column="4">
      <widget class="QPushButton" name="mRightEndBtn">
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="mRightBtn">
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="mPageLabel">
       <property name="text">
        <string>0-0 of 0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="mLeftEndBtn">
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="mCondCombo"/>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="mInputText"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mLeftBtn">
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="mSearchBtn">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mLeftEndBtn</tabstop>
  <tabstop>mLeftBtn</tabstop>
  <tabstop>mRightBtn</tabstop>
  <tabstop>mRightEndBtn</tabstop>
  <tabstop>mCondCombo</tabstop>
  <tabstop>mInputText</tabstop>
  <tabstop>mSearchBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
